app/models/resource_action.rb

```diff
@@ -39,10 +39,10 @@ def automate_queue_hash(target, override_attrs, user, open_url_task_id = nil)
       :open_url_task_id => open_url_task_id,
       :attrs            => attrs,
     }.merge(override_values).tap do |args|
-      args[:user_id] ||= user.id
-      args[:miq_group_id] ||= user.current_group.id
-      args[:tenant_id] ||= user.current_tenant.id
-      args[:username] ||= user.userid
+      args[:user_id] ||= user.try(:id)
+      args[:miq_group_id] ||= user.try(:current_group).try(:id)
```

Comment:
```suggestion
      args[:miq_group_id] ||= user&.current_group_id
```

probably moot since we need to lookup the group to get the tenant but yea.
see if current_tenant_id works